gdpr-compliance-doc.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: GDPR Compliance Documentation
+title: GDPR Compliance and Terms of Service
 ---
 
 # GDPR Compliance Documentation
```

Comment:
```suggestion
# GDPR Compliance and Terms of Service
```

Reply:
Title in the metadata is only for the html metadata but is not the Title that is displayed, this one is

Reply:
I wasn't clear. Sorry.
The idea was to have the two sections inside the same page. Hence not changing the heading level.
So a single `#` to make a GDPR and another for making a Terms of Service section.